marklogic-data-hub-central/ui/src/components/load/load-list.test.tsx

```diff
@@ -62,25 +63,25 @@ describe('Load data component', () => {
     await wait(() => {
         fireEvent.click(saveButton);
     });
-    expect(axiosMock.put).toHaveBeenCalledTimes(1);
+    expect(axiosMock.post).toHaveBeenCalledTimes(1);
 
     //open settings again
 
     await wait(() => {
         fireEvent.click(getByTestId(data.loadData.data[0].name+'-settings'));
     })
-    let targetCollection = getByTitle('customerCollection'); // Additional target collection (Added by user)
+    let targetCollection = getByTitle('addedCollection'); // Additional target collection (Added by user)
 
     expect(getByText('Advanced Settings')).toBeInTheDocument();
     // Check if the settings API is being called.
-    expect(axiosMock.get).toBeCalledWith('/api/steps/ingestion/' + data.loadData.data[0].name + '/settings');
-    expect(getByText('Target Collections:')).toBeInTheDocument();
+    expect(axiosMock.get).toBeCalledWith('/api/steps/ingestion/' + data.loadData.data[0].name);
+    expect(getByText('Target Collections')).toBeInTheDocument();
     expect(targetCollection).toBeInTheDocument(); //Should be available in the document
-    expect(targetCollection).not.toBe(data.loadData.data[0].name); //Should not be same as the default collection
-    expect(getByText('Default Collections:')).toBeInTheDocument();
-    expect(getByTestId(`defaultCollections-${data.loadData.data[0].name}`)).toBeInTheDocument();
-    expect(queryByTitle(data.loadData.data[0].name)).not.toBeInTheDocument();  // The default collection should not be a part of the Target Collection list
-    expect(getByText('Batch Size:')).toBeInTheDocument();
+    expect(targetCollection).not.toBe('testCollection'); //Should not be same as the default collection
```

Comment:
Its better to use variable than string literal for test value.

Reply:
This is data received from the mock endpoint and integrated into the DOM. How do I reference that without it being circular test logic?

Reply:
Right. It is a mock data and so we are trying to include a dataset that matches what we will get back from the server. Hence we had collection name matching the step name. Any collection name would work, but to keep with the response server would return we kept it likewise.

Reply:
But how do I put data retrieved from the mock endpoint into a variable for testing as you are asking?

Reply:
Ok. I was referring to "data.loadData.data[0].name)" as compared to "testCollection".
That way the changes will only be in the test data(in one place) and we wont have to touch the test, if test data changes.